lib/cloud_controller/permissions.rb

```diff
@@ -288,15 +262,15 @@ def can_read_system_environment_variables?(space_guid, org_guid)
       membership.has_any_roles?(ROLES_FOR_SPACE_SECRETS_READING, space_guid, org_guid)
   end
 
-  def untrusted_can_read_route?(space_guid, org_guid)
-    return true if can_read_globally?
+  # def can_read_route?(space_guid, org_guid)
```

Comment:
Is there a reason we've left this method commented out rather than removing it?

Reply:
No, good catch!